### PR TITLE
Major SDK update (QGIS, GDAL, PROJ, sqlite, etc.)

### PR DESCRIPTION
**Pull request by nirvn**
A run of APKs to test things out.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for d9a4cce969957a3c1bf499570f6a9db794897996:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2107-3b1eca3c67936593dc82246ded8ed42f268ec6e5-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2107-3b1eca3c67936593dc82246ded8ed42f268ec6e5-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2107-3b1eca3c67936593dc82246ded8ed42f268ec6e5-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2107-3b1eca3c67936593dc82246ded8ed42f268ec6e5-x86.apk)

File: sdk.conf [hunk 1]
```diff
@@ -1 +1 @@
-osgeo4a_version=20210620
+osgeo4a_version=gdal_up
```
Comment (nirvn) [inline sdk.conf:1]:
```suggestion
osgeo4a_version=20210828
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 3e400561f57d87f4c9a2be4f370a05292fb66d48:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2107-48c0a5cf338f96d61b0f155b6e14815255aa9b32-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2107-48c0a5cf338f96d61b0f155b6e14815255aa9b32-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2107-48c0a5cf338f96d61b0f155b6e14815255aa9b32-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2107-48c0a5cf338f96d61b0f155b6e14815255aa9b32-x86.apk)

**Comment by nirvn** [comment]
Loaded a few projects on my device, everything is good.

In fact, I felt things were slightly faster/smoother. Probably a fiction of my imagination ;) the sqlite update could have unlocked some speed, as well as the micro optimizations done in QGIS.

**Review by m-kuhn — APPROVED** [review]

**Comment by m-kuhn** [comment]
Nice !!

**Pull request merged**